### PR TITLE
car task

### PR DESCRIPTION
**Pull request by VasiliZas**

### CODE REVIEW

**Review by lamaconda — APPROVED** [review]
lgfm, but don't forget about adding empty line in the end of code

File: homework/Vasili_Zasinets/hw1/CarDemo.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
+package com.tutrit.java.quickstart.hw1.car;
+
+public class CarService {
+    private Car car;
+
+    boolean result;
```
Comment (max-java) [inline homework/Vasili_Zasinets/hw1/CarDemo.java:6]: result of what? Please, give meaningful name to the variable.

File: homework/Vasili_Zasinets/hw1/CarDemo.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            car.setCurentSpeed(newSpeed);
+        }
+    }
+
+    public boolean isDriving() {
+        if (car.getCurentSpeed() > 0) {
+            result = true;
+        } else result = false;
+        return result;
+    };
```
Comment (max-java) [inline homework/Vasili_Zasinets/hw1/CarDemo.java:54]:
```
if (car.getCurentSpeed() > 0) {
            return true;
        }
        return false;
```

File: homework/Vasili_Zasinets/hw1/CarDemo.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        this.car = car;
+    }
+
+
+    public void accelerate(int targetSpeed) {
+        while (car.getCurentSpeed() < targetSpeed) {
+            double newSpeed = car.getCurentSpeed() + 1;
+            if (newSpeed > car.getMaxSpeed()) {
+                newSpeed = car.getCurentSpeed();
+            }
```
Comment (max-java) [inline homework/Vasili_Zasinets/hw1/CarDemo.java:26]: this looks like check that could be encapsulate into  method with meaningful name

File: homework/Vasili_Zasinets/hw1/CarDemo.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+            car.setCurentSpeed(newSpeed);
+            if (targetSpeed > car.getMaxSpeed()) {
+                newSpeed = car.getMaxSpeed();
```
Comment (max-java) [inline homework/Vasili_Zasinets/hw1/CarDemo.java:29]: this looks like check that could be encapsulate into method with meaningful name

**Review by max-java — CHANGES_REQUESTED** [review]
please, add some tests and run sonarQube analyse

**Pull request closed without merging**